docs/04-operation-guides/operations/evnt-03-nats-server-system-events.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Acquiring NATS server system account credentials
+---
+## Context
+
+Accessing certain resources in NATS requires [`system_account` privileges](https://docs.nats.io/running-a-nats-service/configuration/sys_accounts). Kyma will automatically generate a `system account` user via a Secret dubbed `eventing-nats-secret` in the `kyma-system` Namespace.
+
+## Prerequisites
+
+None.
+
+## Procedure
+
+To acquire the cretentials run the following command:
+
+```bash
+kubectl get secrets -n kyma-system eventing-nats-secret -oyaml | grep -e accountsJson | awk '{print $2}' | base64 -d | grep {user: | awk '{$1=$1};1' | awk '{print substr($0, 2, length($0) - 2)}'
+```
+
+### Result: 
+This will return the credentials for the `system account` user.
+```bash
+user: admin, password: <your password>
+```
+### Next steps:
+To access the NATS server with the [nats-cli tool](https://github.com/nats-io/natscli) you need to first forward its port:
+```bash
```

Comment:
need to add 3 blank spaces at the beginning to adjust the indentation because I added the step number.

```suggestion
   ```bash
```